.storybook/3.COLORS.stories.mdx

```diff
@@ -0,0 +1,182 @@
+import { Meta } from '@storybook/addon-docs';
+import ActionaleMessage from '../ui/components/ui/actionable-message';
+import designTokenDiagramImage from './images/design.token.graphic.svg';
+
+<Meta title="Design Tokens / Color" />
+
+# Color
+
+Color is used to express style and communicate meaning.
+
+<ActionaleMessage
+  type="warning"
+  message="We are in the process of consolidating all of our colors, making them accessible and enabling theming. Many of the colors used throughout the codebase are deprecated please follow the guide below to ensure you are using the correct colors when building MetaMask UI"
+/>
+
+<br />
+
+## Design Tokens
+
+We are importing design tokens as CSS variables from [@metamask/design-tokens](https://github.com/MetaMask/design-tokens) repo to help consolidate colors and enable theming across all MetaMask products.
+
+### Token Tiers
+
+We follow a 3 tiered system for color design tokens and css variables.
+
+<div
+  style={{
+    textAlign: 'center',
+    backgroundColor: 'var(--color-background-alternative)',
+    padding: 32,
+  }}
+>
+  <img width="80%" src={designTokenDiagramImage} />
+</div>
+
+<br />
+<br />
+
+### **Brand colors** (tier 1)
+
+These colors **SHOULD NOT** be used in your styles directly they are used as a reference for the [Theme Colors](#theme-colors-tier-2). Brand colors should just keep track of every color used in our app.
```

Comment:
Thanks @digiwand! I played around with a few different ways of structuring the information and it's definitely not as optimal as it could be. For V1 I'm wondering if this will do for now and add to the docs as questions arise? I can then improve docs based on those questions?

Reply:
Works with me 👍